### ISSUE TITLE
[BUG] Temporal detections are not rendered in App or draw_labels()

### ISSUE THREAD

**Issue by brimoor** [voxel51/fiftyone#2430]
`TemporalDetection` and `TemporalDetections` instances are not currently rendered in the App or by `draw_labels()`:

```py
import fiftyone as fo
import fiftyone.zoo as foz

dataset = foz.load_zoo_dataset("quickstart-video", max_samples=1)
dataset.compute_metadata()

# Classifications
dataset.set_values(
    "classifications",
    [fo.Classification(label="test1") for _ in range(len(dataset))],
)

# Temporal detection
dataset.set_values(
    "detection",
    [
        fo.TemporalDetection(label="test2", support=[10, fc - 10])
        for fc in dataset.values("metadata.total_frame_count")
    ]
)

# Temporal detections
dataset.set_values(
    "detections",
    [
        fo.TemporalDetections(
            detections=[
                fo.TemporalDetection(label="test3", support=[10, fc - 10])
            ]
        )
        for fc in dataset.values("metadata.total_frame_count")
    ]
)

# Temporal detections are not rendered in the App
session = fo.launch_app(dataset)

# Temporal detections are not included in rendered output
dataset.draw_labels("/tmp/quickstart-video")
```

**Comment by ehofesmann** [comment]
@brimoor @lanzhenw I believe this has been resolved (at least for rendering in the App). Can you close this if that's true?

**Comment by brimoor** [comment]
This has been resolved as of `fiftyone==0.20`.
